distribution/src/main/resources/config/elasticsearch.yml

```diff
@@ -7,8 +7,7 @@
 # The primary way of configuring a node is via this file. This template lists
 # the most important settings you may want to configure for a production cluster.
 #
-# Please see the documentation for further information on configuration options:
-# <https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html>
+# Please consult the documentation for further information on configuration options.
```

Comment:
I considered that but we shouldn't link to current because if someone is looking at a 5.x installation after the 6.0 release, current will take them to the wrong place (or even between minors).